src/selectors/entities/users.test.js

```diff
@@ -458,7 +458,19 @@ describe('Selectors.Users', () => {
         // Should show full name since preferences is being used and LockTeammateNameDisplay is false
         assert.deepEqual(Selectors.makeGetDisplayName()(newTestState, testUser1.id), 'First Last');
 
-        newTestState.entities.general.config.LockTeammateNameDisplay = 'true';
+        newTestState = {
```

Comment:
Can you elaborate on this change?

Reply:
Talked with Hossein offline, and concluded this is necessary due to memoization: Redux was caching the results here.

Reply:
I have updated the way I'm doing this to make it cleaner. Thanks a lot for clarifying why this is the correct solution! Appreciate it.